Update access and permissions (#91)
abtion/guidelines: tools_and_services/access_and_permissions.md

You can request access to vaults through Slack #access-and-permissions.

## Google groups

> - If it's needed a google group can be set up for the client, e.g. `danish-shipping@abtion.com`.

Reviewer: We should set it as a standard. 
```suggestion
- A google group will be set up per client, e.g. `danish-shipping@abtion.com`.
```

> - Client groups can be used for creating client specific accounts for github, azure etc.

Reviewer: ```suggestion
- The group will be used for creating client-specific accounts for services such as github, azure, etc.
```

## Github

- Access is controlled through teams. Abtioneers are part of teams. Teams have access to repos. E.g. Wordpress team will have access to several repos.
- To give a better overview, members should not get granted permissions directly to any repo. 
- The Abtion team has read access to all the apps. All Abtioneers are members of the Abtion team.
- Every new client with a team will have their own team. E.g. `danish-shipping`

> - For rails projects a client specific github account will be created - using the team's google group. This account must have admin-access the repository.

Reviewer: Why do we mention Rails?

Author: Probably not needed
